openpype/plugins/publish/collect_slate_global.py

```diff
@@ -0,0 +1,76 @@
+import os
+import pyblish.api
+from openpype.settings import get_current_project_settings
+from openpype.settings.lib import get_anatomy_settings
+from openpype.pipeline.context_tools import get_template_data_from_session
+
+
+
+class CollectSlateGlobal(pyblish.api.InstancePlugin):
+    """
+    Check if slate global is active and enable slate workflow for
+    selected families
+    """
+    label = "Collect for Slate Global workflow"
+    order = pyblish.api.CollectorOrder + 0.499
+    families = [
+        "review",
+        "render"
+    ]
+
+    _slate_settings_name = "ExtractSlateGlobal"
+
+    def process(self, instance):
+
+        publ_settings = get_current_project_settings()\
+            ["global"]["publish"]
+        version_padding = get_anatomy_settings(
+            os.environ.get("AVALON_PROJECT")
+        )["templates"]["defaults"]["version_padding"]
```

Comment:
```suggestion
        context = instance.context
        publ_settings = context.data["project_settings"]["global"]["publish"]
        version_padding = context.data["anatomy"].templates["defaults"]["version_padding"]
```

I suspect that something like this should also work. Not sure about that anatomy - but these types of settings should've already been collected.